src/menu-button.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const React = require('react');
+const classnames = require('classnames');
+const Icon = require ('./icon');
+
+const baseClasses = {
+  'mdl-button': true,
+  'mdl-js-button': true,
+  'mdl-button--icon': true
+};
+
+class MenuButton extends React.Component {
+  constructor(...args){
+    super(...args);
+    this._autoId = '_' + Math.random().toString(36).slice(2);
+
+  }
+  render() {
+    const {
+      className,
+      children
```

Comment:
you can probably use default assignment to assign the default icon to `children` here